addons/hr_expense/tests/test_expenses.py

```diff
@@ -214,6 +214,30 @@ def test_expense_values(self):
             },
         ])
 
+        self.assertRecordValues(expense_sheet, [{'total_amount': 1500.0, 'untaxed_amount': 1304.35, 'total_amount_taxes': 195.65}])
+        self.assertTaxTotals(expense_sheet.account_move_id, {
+            'amount_total': 2500, # why not --> expense_sheet.total_amount?
+            'amount_untaxed': 2173.92, # why not --> expense_sheet.untaxed_amount?
```

Comment:
This is confusing for me :thinking: 

Like here amount_total is sum of amounts in different currencies. Should it be like so?

For a better explanation of the problem -https://www.odoo.com/web#cids=1&menu_id=4720&action=333&active_id=1251&model=project.task&view_type=form&id=3049094